### PR TITLE
Leatherworking improvements

### PR DESCRIPTION
**Pull request by MadmanMartian**
The leather you use now matters slightly, as it names the final object.

### Leather recipes added
 - Cowboy hat
 - Leather botanist gloves
 - Leather shoes
 - Leather Satchel
 - Leather Wallet
 - Leather armor (Hat and suit, apply different hides to give it different properties)
### Requests
- [x] Human skin hide sheet sprite - Human skin hide is in the code, but it lacks a sprite and an acquisition form. If I had a sprite, I'd add it to human butchering. Courtesy of @Blithering 
- More leatherwork recipe ideas.

:cl:
* rscadd: More leather recipes added.
* rscadd: The crafted leather items now inherit the name of the source of the leather. Corgi leather cowboy hats, Cat leather shoes, et cetera.

### CODE REVIEW

**Comment by AutismSpider** [comment]
Can you add leather bag and coat?

**Comment by MadmanMartian** [comment]
leather satchel can be done, yes.

Coat would require a sprite. It's another thing in the code without an icon.

**Comment by Jubisloviu** [comment]
Leather Jackets would be nice
Leather Armor maybe?, imagine styling a xenomorph leather armor on hallways!
Gimp Suits
Leather Belts, not tool-belts, just normal belts for fashion purposes
Fingerless Leather gloves.
that Van Darkholme mask

**Comment by Diavill** [comment]
leather belts would be great to fulfill the true CIA cosplay set

**Comment by yclatious** [comment]
Leather boots would be nice

**Comment by strawberrytan** [comment]
Vox skin tajaran skin and rare premium unathi skin when

**Comment by small-boss** [comment]
furcoats when

**Comment by Riekland** [comment]
Could the darkholme suit not work as a generic leather thing sprite

**Comment by Blithering** [comment]
Xenomorph hide gear was part of snowmap, there might be something salvageable from that

**Comment by N3X15** [comment]
SINCE WHEN DID WE HAVE AN ICON DIFF BOT

WHAT THE HELL

**Comment by DamianX** [comment]
SINCE THE DAY I ASKED TG TO SHARE, NOW CALM DOWN

File: code/modules/clothing/head/leather.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	siemens_coefficient = 0.6
+
+/obj/item/clothing/head/leather/attackby(obj/W, mob/user)
+	if(istype (W, /obj/item/stack/sheet/animalhide))
+		var/obj/item/stack/sheet/animalhide/S = W
+		if(S.amount < 2)
+			return ..()
+		user.visible_message("<span class='notice'>[user] starts installing \the [S] into and onto \the [src].</span>", \
+		"<span class='notice'>You start installing \the [S] into \the [src].</span>")
+		if(do_after(user, src, 40) && S.use(2))
```
Comment (DamianX) [inline code/modules/clothing/head/leather.dm]: `4 seconds`

**Comment by DamianX** [comment]
changelog so that we can be good to go?

**Pull request merged**